.travis.yml

```diff
@@ -1,30 +1,32 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
 
 env:
-  - SYMFONY_VERSION=2.5.*
-  
+  - SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+
 matrix:
-  allow_failures:
-    - env: SYMFONY_VERSION=dev-master
   include:
+    - php: 5.3
+      env: SYMFONY_VERSION=2.3.* COMPOSER_FLAGS="--prefer-lowest"
     - php: 5.5
       env: SYMFONY_VERSION=2.3.*
     - php: 5.5
-      env: SYMFONY_VERSION=2.4.*
+      env: SYMFONY_VERSION=2.6.*
     - php: 5.5
-      env: SYMFONY_VERSION=dev-master
-    
-
+      env: SYMFONY_VERSION=2.7.* SYMFONY_DEPRECATIONS_HELPER=weak
+    - php: 5.5
+      env: SYMFONY_VERSION=2.8.* SYMFONY_DEPRECATIONS_HELPER=weak
 
-before_script:
-  - composer self-update
-  - echo 'memory_limit = -1' >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini
-  - composer require symfony/symfony:${SYMFONY_VERSION} --prefer-source
+before_install:
+  - composer self-update || true
+  - sh -c 'if [ "${TRAVIS_PHP_VERSION}" != "hhvm" ]; then echo "memory_limit = -1" >> ~/.phpenv/versions/$(phpenv version-name)/etc/conf.d/travis.ini; fi;'
+  - composer require --no-update symfony/symfony:${SYMFONY_VERSION}
+  - COMPOSER_ROOT_VERSION=dev-master composer update $COMPOSER_FLAGS --prefer-source --no-interaction
   - vendor/symfony-cmf/testing/bin/travis/phpcr_odm_doctrine_dbal.sh
```

Comment:
honestly I just wanted the PR to be rebased and the install part to work .. 
lets do all of the above things in a new PR, ok?